internal/incident/incident.go

```diff
@@ -533,6 +481,74 @@ func (i *Incident) RestoreEscalationStateRules(states []*EscalationState) {
 	}
 }
 
+// getEscalationRecipientsChannel returns the configured channel of all the provided escalation recipients.
+func (i *Incident) getEscalationRecipientsChannel(escalation *rule.Escalation, t time.Time) recipientsChannelType {
+	if i.Recipients == nil {
+		i.Recipients = make(map[recipient.Key]*RecipientState)
+	}
```

Comment:
I don't see why this function initializes `i.Recipients` given that it never writes to that map anyways.